### DISCUSSION TITLE
Do not add karma to Lemmy

### DISCUSSION THREAD

**Discussion by gylotip** [LemmyNet/lemmy-ui#1549] [General]
Hello, so I saw [a comment where they discovered user score tracking](https://lemmy.ml/comment/907243), which I really do not want to be added to Lemmy. User karma ruins discussions, and people will farm karma and be afraid of losing karma. This will reduce the quality of discussions. I say it again, please, do not add karma to Lemmy. We do not want to farm karma, and communities will use karma to restrict users. This is a bad idea, so please, do not add karma to Lemmy. Kbin already has it, so Lemmy doesn't need one too.

**Comment by Kommynct** [comment]
as far as i'm aware that only exists because instances are transparent and administrators need that for moderation.

It's foss software, they aren't going to add anything malicious, and if such a thing is added it would be possible to disable by each instance.

**Comment by alectrocute** [comment]
This is not a feature request. Please carefully read the requirements before creating a new issue here. Thank you!
